src/main/resources/com/mathworks/ci/UseMatlabVersionBuildWrapper/help.html

```diff
@@ -1,3 +1,9 @@
-<p>
-Specify the MATLAB version for this build.
-</p>
\ No newline at end of file
+<div>
+    <p>
+        Use this list to specify the MATLAB version for the build. The plugin prepends your specified MATLAB to the PATH environment variable and uses it to perform the build. <br>
+        The list includes the names of all MATLAB versions that have been registered as a tool in <u>Jenkins</u>. Select one of the tools, or select <b>Custom…</b> if you want to specify a different version. When you select <b>Custom…</b>, you must enter the full path to the desired MATLAB root folder in the <b>MATLAB root</b> box.
+    </p>
+    <p><br>
+        <b>Note:</b> In multi-configuration projects, the values specified by <b>Use MATLAB Version</b> value takes precedence over the values specified by the <b>MATLAB</b> axis.

```

Comment:
Please make this change:

by <b>Use MATLAB Version</b> value takes ---> 
by <b>Use MATLAB version</b> takes

Details:
-To match the UI, "v" in version should not be capitalized.
-"value" after Use MATLAB version doesn't make sense.

Reply:
Done.